tests/src/python/test_qgsserver_wms_getfeatureinfo.py

```diff
@@ -255,6 +255,44 @@ def testGetFeatureInfoRelationReference(self):
                                  'wms_getfeatureinfo-values2-text-xml',
                                  'test_project_values.qgz')
 
+    def testGetFeatureInfoSortedByDesigner(self):
+        """Test GetFeatureInfo resolves DRAG&DROP Designer order when use attribute form settings for GetFeatureInfo
+        is checked, see https://github.com/qgis/QGIS/pull/41031
+        """
+        mypath = self.testdata_path + "test_project_values.qgz"
+        self.wms_request_compare('GetFeatureInfo',
+                                 '&layers=layer2&styles=&' +
+                                 'VERSION=1.3.0&' +
+                                 'info_format=text%2Fxml&' +
+                                 'width=926&height=787&srs=EPSG%3A4326' +
+                                 '&bbox=912217,5605059,914099,5606652' +
+                                 '&CRS=EPSG:3857' +
+                                 '&FEATURE_COUNT=10' +
+                                 '&WITH_GEOMETRY=True' +
+                                 '&QUERY_LAYERS=layer3&I=487&J=308',
+                                 'wms_getfeatureinfo-values5-text-xml',
+                                 'test_project_values.qgz')
+
+    # TODO make the GetFeatureInfo Work with Order by Drag&DROp Designer when using joined layer
```

Comment:
Can you remove this block?

Reply:
Hi and thanks for reviewing.

from 273:276 or just the ``# TODO`` comment?

If I remove the 274 line the test doesn't work. I copied the template from the other tests.

Reply:
I mean the commented part, is there a good reason to leave it there?

Reply:
It's a test that shows a bug (I wrote in the respective commit).
If sort order is enabled on a joined layer, a GetFeatureInfo request will crash the server

The error is:
```
ASSERT failure in QVector<T>::operator[]: "index out of range", file /usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h, line 458
```

Last time when I discovered server bugs you said it would be great if I could add a server test that shows the failing request.

The commented part it's exactly that. However, due to the crash it seems the normal adding a ``@unittest.expectedFailure`` doesn't work so I commented it.

Should I remove the failing commented test?

Reply:
Yes, I think that you could add the test in the crash bug report.

Reply:
Done! Thanks for the guidance!